testnets/bitcannadev2/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "bitcannadev2",
+    "assets": [
+        {
+            "description": "The BCNA coin is the transactional token within the BitCanna network, serving the legal cannabis industry through its payment network, supply chain and trust network.",
+            "denom_units": [
+                {
+                    "denom": "ubcna",
+                    "exponent": 0
+                },
+                {
+                    "denom": "bcna",
+                    "exponent": 6
+                }
+            ],
+            "base": "ubcna",
+            "display": "bcna",
+            "name": "BitCanna",
+            "symbol": "BCNA",
+            "logo_URIs": {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/bitcanna-dev-6/images/bcna.png",
```

Comment:
oh, looks like it still fails validation. I think it's because of the locations here are incorrect. Can you correct the folder name on these images please? (\bitcanna-dev-6\images... -> \bitcannadev2\images...)